api/kyverno/v1/common_types.go

```diff
@@ -350,6 +354,10 @@ func (m *Mutation) SetPatchStrategicMerge(in apiextensions.JSON) {
 	m.RawPatchStrategicMerge = ToJSON(in)
 }
 
+func (m *Mutation) IsMutateExistingOnPolicyUpdate() *bool {
```

Comment:
What's that ?

Reply:
It is a function that returns the value of `mutateExistingOnPolicyUpdate` under the mutate rule.

Reply:
Why not access the field directly ?

Reply:
We can access it directly but we do have the same method for the generate rule so I thought it would be better to have the same for mutation.